sdk/core/azure-core-test/src/main/java/com/azure/core/test/InterceptorManager.java

```diff
@@ -191,8 +193,11 @@ private File getRecordFile(String testName) {
         }
 
         String filePath = folderFile.getPath() + "/" + testName + ".json";
```

Comment:
Instead of generating the file path, checking existence, failing or get the file why not use

```java
File playbackFile = new File(folderFile, testName + ".json");
if (!playbackFile.exists()) {
    // throw exception
}

return playbackFile;
```